deis/monitor
# chore(grafana): Update dashboards for new telegraf

Tags for data collected by the kubernetes changed after the plguin was merged. So we needed to update the dashboards.

Test Steps:
- clone pr
- `cd telegraf && make build push upgrade`
- `cd ../grafana && make build push upgrade`
- open browser and goto `http://grafana.mydomain.com` login using `admin/admin`
- The dashboards should have memory and cpu information

It is also useful to verify that we are not leaking connections with this PR as it will contain the new telegraf binary that contains the fix. 

To check for that please ssh onto one of your worker nodes that is running the telegraf daemonset and run the following command `netstat -tan | grep 10255` you should only see 1 or 2 connections open and it should never grow.
